code/_globalvars/traits.dm

```diff
@@ -83,6 +83,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_ABDUCTOR_SCIENTIST_TRAINING" = TRAIT_ABDUCTOR_SCIENTIST_TRAINING,
 		"TRAIT_SURGEON" = TRAIT_SURGEON,
 		"TRAIT_STRONG_GRABBER" = TRAIT_STRONG_GRABBER,
+		"TRAIT_STRONG_PUNCHER" = TRAIT_STRONG_PUNCHER,
```

Comment:
shouldnt this be accurate puncher or something

Reply:
Yeah ryll brought that up actually. I'll change it to a more accurate trait name (haha get it)